cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -395,6 +410,10 @@ func main() {
 		projList = gcloudProject()
 	}
 
+	if envInstances := os.Getenv("INSTANCES"); envInstances != "" {
+		instList = append(instList, stringList(envInstances)...)
```

Comment:
It seems a bit weird to join the environment variable with the values from the command line. Are their examples of other common tools with this behavior. IIRC most things have the command line value override the environment variable.

Reply:
yep, fixed.